cpu/esp32/periph/rtc.c

```diff
@@ -20,11 +20,12 @@
  */
 
 /*
- * If RTC_TIMER_USED is 0, the microsecond system timer is used to emulate an
- * RTC, otherwise the RTC timer is used. Advantage of using RTC over sytem
- * timer is that it also continues in deep sleep and after software reset.
+ * If module esp_rtc_timer is enabled, the 48-bit RTC hardware timer is used
+ * directly. Otherwise the PLL driven 64-bit microsecond system timer is used
+ * to emulate a RTC timer (default). This emulated RTC timer results into much
+ * better accuracy. The Advantage of using RTC hardware timer over sytem timer
+ * is that it would also continue in deep sleep mode.
```

Comment:
> The Advantage of using RTC hardware timer over sytem timer is that it would also continue in deep sleep mode.

Usually that's the whole point of having a RTC :wink:

Reply:
Yes, of course. But with these changes, the much more accurate system timer is used. RTC is only used to restore the system time after reboot from deep sleep.